pkg/kubevirt/machine.go

```diff
@@ -237,6 +237,25 @@ func (m *Machine) IsReady() bool {
 	return m.hasReadyCondition()
 }
 
+// IsLiveMigratable reports back the live-migratability state of the VM
+func (m *Machine) IsLiveMigratable() (bool, string, error) {
+	if m.vmiInstance == nil {
+		return false, "", fmt.Errorf("VMI is nil")
+	}
+
+	for _, cond := range m.vmiInstance.Status.Conditions {
+		if cond.Type == kubevirtv1.VirtualMachineInstanceIsMigratable {
+			if cond.Status == corev1.ConditionTrue {
+				return true, "", nil
+			} else {
+				return false, cond.Message, nil
+			}
+		}
+	}
```

Comment:
in addition to the message, would it also make sense to pass the original "Reason" back from the VMI condition and copy it to the KubeVirtMachine reason?

Reply:
done. the reason from the VMI condition is now being copied to the KubevirtMachine condition as-is.